pom.xml

```diff
@@ -337,7 +337,9 @@
                 <redirectTestOutputToFile>true</redirectTestOutputToFile>
                 <forkCount>4</forkCount>
                 <excludes>
-                    <exclude>**/*PerformanceTest.java</exclude>
+                    <exclude>**/*PerformanceTest.java</exclude>					
+                    <exclude>**/*MEFExporterIntegrationTest.java</exclude>
+                    <exclude>**/*LuceneSearcherPresentTest.java</exclude>
```

Comment:
DO you think this is a better way than putting a @Ignore on the tests to ignore?  Seems to me that it might make it hard to figure out why the tests aren't running